message_tagging_service/consumer.py

```diff
@@ -29,23 +30,32 @@
 logger = logging.getLogger(__name__)
 
 
-class RHMessage(object):
-    """Representing a message received from rhmsg message bus"""
+class UMBMessage(object):
+    """Representing a message received from rhmsg message bus
 
-    def __init__(self, raw_msg):
-        self._raw_msg = raw_msg
+    :param msg: the message object received from underlying rhmsg message bus.
+    :type msg: ``proton.Message``
+    """
+
+    def __init__(self, msg):
+        self._orig_msg = msg
+
+        if msg.properties['content-type'] == 'text/json':
```

Comment:
There's also `application/json`. I wonder if it would just always load it as Json. In the case of invalid json, `ValueError` is raised with some context. I'm not sure what we gain by raising `NotImplementedError` instead.